ansible/roles/ocp4-workload-ai-ml-workflows-demo/tasks/per_user_remove_workload.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Implement your Workload deployment tasks here
+
+- debug:
+    msg: "Removing ML Workflows workshop materials from preexisting project {{ project_name }}"
+
+- name: Create the custom notebook image
+  k8s:
+    state: absent
+    namespace: "{{ project_name }}"
+    definition: "{{ item }}"
+  with_items:
+    - "{{ lookup('template', 'custom-notebook-imagestream.yaml.j2') }}"
+
+- name: Create the pipeline builder image
+  k8s:
+    state: absent
+    namespace: "{{ project_name }}"
+    definition: "{{ item }}"
+  with_items:
+    - "{{ lookup('template', 'pipeline-s2i-imagestream.yaml.j2') }}"
```

Comment:
Same recommendation here, `with_items` to `loop` and move template to `definition`.